Products overhaul (#77)
pretix/pretix-docs: docs/guides/products/index.md

> # Products

Reviewer: Hier muss ich noch mal drüber, aber bevor du den Rest auch nicht hast ...

A product is anything sold via pretix: tickets, gift cards, conference t-shirts and so on. 
pretix offers you almost unlimited possibilities for configuring and structuring products. 
This article guides you through the basic process of creating a product and explains several practical applications of some of the more advanced features of pretix. 

## Prerequisites

Products are configured on the event level, so you have to create an event first. 

## General usage

This section guides you through the basic process of product creation. 
This involves first creating categories, then the products themselves, and finally quotas. 
You cannot create a product without choosing a category for it. 
You cannot create a quota without adding at least one product to it. 
Thus, this guide will explain those steps in that order.

### Creating and editing categories

![Page titled 'Product categories', showing a list of categories only containing 'Tickets' and a button for creating a new category.](../../assets/screens/products/categories.png "Product categories screenshot") 

> Categories separate standalone products from additional products. 

Reviewer: > Categories separate standalone products from additional products. 

1. An dieser Stelle wird nicht wirklich klar, was standalone products sind und noch weniger, was additional products sind. (Weiter unten, wo "standalone" in der Erklärung für normal products verwendet wird, hingegen schon, da der Kotext dort benötigte Informationen liefert.) Zudem ist die Entscheidung ein bisschen diffus, oder? (Typ "normal+cross-selling" ...)
2. Ist das wirklich ihre Hauptfunktion? Damit einzuleiten, impliziert das imo.
3. Ich würde explizit machen, dass es auch mind. 1 category braucht, wenn es nur gesamt 1 category hat (weil alle Produkte in die gleiche category gehören).

> If you want to sell not only admission tickets, but also extras such as stickers, you have to have at least two different categories. 

Reviewer: > If you want to sell not only admission tickets, but also extras such as stickers, you have to have at least two different categories. 

Das (stickers) ist ein ziemlich spezifisches Beispiel; haben wir noch ein anderes?

You also need an extra category if you are planning to use the cross-selling feature. 

> Sorting products into categories can help you better keep track of them in the backend. 

Reviewer: ```suggestion
Sorting products into categories can help you keep track of them in the backend. 
```

> Finally, your shop page will display products grouped by categories which can help customers find the article they are looking for more easily. 

Reviewer: ```suggestion
Finally, your shop page will display products grouped by categories which can help customers find the item they are looking for more easily. 
```

In order to edit categories, navigate to :navpath:Your event → :fa3-ticket: Products → Categories:. 
This page shows the list of all product categories. 

Click the :btn-icon:fa3-plus: Create a new category: button and give the new category a descriptive name. 
Choose the category type depending on the type of products in this category: normal, add-on, cross-selling, or normal + cross-selling. 

Normal products are standalone products that can be purchased directly. 
Add-on products are products that are only offered as add-ons to normal products. 
Cross-selling products are products that are only offered in the cross-selling step as a customer is purchasing other products. 
Finally, products in the "normal + cross-selling" category are both offered as standalone products and in the cross-selling step. 

> The "Cross-selling condition" setting below determines how products in cross-selling and normal + cross-selling categories are offered in your shop. 

Reviewer: ```suggestion
The "Cross-selling condition" setting below determines how products in the cross-selling and normal + cross-selling categories are offered in your shop. 
```

Every product has to be part of at least one quota before it becomes available in the shop. 

If you want to create or edit quotas, navigate to :navpath:Event → :fa3-ticket: Products → Quotas:. 
This page shows the list of all quotas for the event as well as the total capacity and how many items are left for each quota. 

Click the :btn-icon:fa3-plus: Create a new quota: button. 
Choose a name and a capacity, check the products to be contained in this quota, and any advanced options, and then click the :btn:Save: button. 

You can also edit an already existing quota by clicking its name or the :btn-icon:fa3-edit:: edit button next to it in the list. 

## Applications 

This section covers advanced use cases and how to implement them using the options pretix offers for customizing products. 

### Time slots 

A more advanced use case of pretix is using pretix for time-slot-based access to an area with a limited visitor capacity, such as a museum or other attraction. 
This guide will show you the quickest way to set up such an event with pretix.

First of all, when creating your event, you need to select that your event represents an "event series":

> ![](../../assets/screens/products/create_step11.png "Create step 11")

Reviewer: > ![](../../assets/screens/products/create_step11.png "Create step 11")

Uninformative und nonexistente Bildbeschreibung ist leider uninformativ und nonexistent.
Welcher der beschriebenen Schritte ist der 11.?

For general instructions on how to set up an event series with pretix, refer to our guide on [event series](../event-series.md).

#### Creating slots

To create the time slots, you need to create a number of "dates" in the event series. 
Navigate to :navpath:Your event → :fa3-calendar: Dates: and click the :btn-icon:fa3-plus:Create many new dates: button. 
Then, first enter the pattern of your opening days. 
In the example, the museum is open week Tuesday to Sunday. 
We recommend to create the slots for a few weeks at a time, but not e.g. for a full year, since it will be more complicated to change things later.

> ![](../../assets/screens/products/timeslots_create1.png "Timeslots create 1")

Reviewer: > ![](../../assets/screens/products/timeslots_create1.png "Timeslots create 1")

Uninformative und nonexistente Bildbeschreibung ist leider uninformativ und nonexistent.
Welches ist der erste Schritt, der hier zu sehen ist?

Then, scroll to the times section and create your time slots. 
You can do any interval you like. 
If you have different opening times on different days of the week, you will need to go through the creation process multiple times.

> ![](../../assets/screens/products/timeslots_create_21.png "Timeslots create 21")

Reviewer: > ![](../../assets/screens/products/timeslots_create_21.png "Timeslots create 21")

Uninformative, nonexistente Bildbeschreibung ist leider uninformativ und nonexistent - und verwirrend: Schritt 2.1.? Schritt 21? So oder so findet sich die Nummerierung nicht im Text.

Scroll further down and create one or multiple quotas that define how many people can book a ticket for that time slot. 
In this example, 50 people in total are allowed to enter within every slot:

> ![](../../assets/screens/products/timeslots_create_31.png "Timeslots create 31")

Reviewer: > ![](../../assets/screens/products/timeslots_create_31.png "Timeslots create 31")

Selbiges, nur mit 3.1./31. statt 2.1./21. ...

Do **not** create a check-in list at this point. 
We will deal with this further below in the guide. 

> Now, press "Save" to create your slots.

Reviewer: > Now, press "Save" to create your slots.

Folgt das absichtlich nicht unserer üblichen Formatierung für Knöpfe?

!!! Warning

    If you create a lot of time slots at once, the server might need a few minutes to create them all in our system. 
    If you receive an error page because it took too long, please do not try again immediately but wait for a few minutes. 
    Most likely, the slots will be created successfully even though you saw an error.

#### Event settings

> We recommend that you navigate to :navpath:Your Event → :fa3-wrench: Settings → General:. 

Reviewer: > We recommend that you navigate to :navpath:Your Event → :fa3-wrench: Settings → General:. 

... ist das wirklich nur eine **Empfehlung** oder müssen Kund*innen nicht faktisch dort entlang, wenn sie die Einstellungen bearbeiten wollen?

Open the :btn:Display: tab and set "Default overview style" to "Week calendar":

> ![](../../assets/screens/products/timeslots_settings_11.png "Timeslots settings 11")

Reviewer: > ![](../../assets/screens/products/timeslots_settings_11.png "Timeslots settings 11")

Bildbeschreibung fehlt.

Now, your ticket shop should give users a nice weekly overview over all time slots and their availability:

> ![](../../assets/screens/products/timeslots_presale1.png "Timeslots presale 1")

Reviewer: > ![](../../assets/screens/products/timeslots_presale1.png "Timeslots presale 1")

Bildbeschreibung fehlt.

#### Check-in

If you want to scan tickets at the entrance to your event and only admit the ticket holders at their designated time, we recommend the following setup: 
Go to "Check-in" in the main navigation on the left and create a new check-in list. 

> Give it a name and do not choose a specific data. 

Reviewer: ```suggestion
Give it a descriptive name and do not choose a specific date. 
```

We will use one check-in list for all dates. 
Then, go to the "Advanced" tab at the top and set up two restrictions to make sure people can only get in during the time slot they registered for. 

Under "Custom check-in rule", click the :btn-icon:fa3-plus-circle: Add condition: button and select "All of the conditions below (AND)" from the dropdown menu. 
Click :btn-icon:fa3-plus-circle: Add condition: again and select "Current date and time", then "is after", then "Event start". 
Leave the "Tolerance (minutes)" field empty. 
Click :btn-icon:fa3-plus-circle: Add condition: again and select "Current date and time", then "is before", then "Event end". 
Again, leave the "Tolerance (minutes)" field empty. 
Your custom check-in rule should now look like the one in the following screenshot: 

![Custom check-in rule page set up according to the instructions above.](../../assets/screens/products/timeslots_checkinlists1.png "Timeslots check-in lists 1")

If you want, you can enter a tolerance of e.g. "10" if you want to be a little bit more relaxed and admit people up to 10 minutes before or after their time slot.

> Now, download our Android or Desktop app and register it to your account. 

Reviewer: ```suggestion
Now, download our Android or Desktop app and connect it to your account. 
```

So als erste Verbesserung mal. 
Es stellen sich Folgefragen wie "welche App", die vielleicht besser mit Existenz eines entsprechenden Artikels beantwortet werden sollten.

The app will ask you to select one of the time slots, but it does not matter, you can select any one of them and then select your newly created check-in list. 
That's it, you're good to go!

### Season tickets

Season tickets and similar time-based tickets are popular for swimming pools, sports clubs, theaters and lots of other types of venues. 
In this section, we show you different ways to set them up with pretix. 
Of course, other types of tickets such as week tickets, month tickets or tickets of ten can be created with the same mechanism.

> There is a big difference between the two ways we show below.

Reviewer: > There is a big difference between the two ways we show below.

Würde die beiden Methoden erst benennen und dann ihre Unterschiede darlegen.

With Option A, a customer who purchases a season ticket creates an account with their email address and a password and the season ticket will be saved in that account. 
If the customer wants to use the season ticket, they need to buy an additional free ticket for the specific event they want to visit. 
This makes sense for all events or venues with limited capacity or reserved seating, because it still allows you to set an upper limit of people showing up for a specific event or time slot.

With Option B, a customer who purchases a season ticket receives a single ticket with a single QR code that can be used an unlimited number of times. 
This makes sense if the capacity of your venue is virtually unlimited and you do not need to know in advance how many season ticket holders will show up.

#### Option A: Memberships and multiple tickets

Since this approach requires customers to be identified with a customer account, you first need to enable the customer accounts feature in your organizer settings in the "Customer accounts" tab. 
See also: [Customer accounts](../customer-accounts.md) 

> ![](../../assets/screens/products/seasontickets_orgsettings1.png "Season tickets organizer settings 1")

Reviewer: > ![](../../assets/screens/products/seasontickets_orgsettings1.png "Season tickets organizer settings 1")

Nonexistente Bildbeschreibung ist nonexistent.

After doing so, a new menu item "Customer accounts" will also show up in the main menu of your organizer account on the left. 
Open its menu and click "Membership types". 
Then, select to "create a new membership type".

You can name the membership type in a way that clearly explains where it is valid, e.g. "season pass main location" or "season pass all locations". 
There are a few details you can configure on this page, such as whether the season pass can be used by multiple different persons, or if the season pass can be used for multiple tickets for the same time slot. 
You can also define a maximum number of usages, which is useful if you e.g. use this feature to add a "ticket of ten".

> ![](../../assets/screens/products/seasontickets_membershiptype1.png "Season tickets membership type 1")

Reviewer: > ![](../../assets/screens/products/seasontickets_membershiptype1.png "Season tickets membership type 1")

Bildbeschreibung fehlt

Next, you need a way of selling these season passes. 

> Theoretically this can be done through the same event series that you usually use, but it's probably cleaner and easier to find for customers if you create a new event that you only use to sell season passes. 

Reviewer: > Theoretically this can be done through the same event series that you usually use, but it's probably cleaner and easier to find for customers if you create a new event that you only use to sell season passes.

"probably" ist ein bisschen schwach. 

> While it is possible to do this through [...], creating a new event only for season passes keeps things clearer for you and your customers.

o.ä.

> The start and end date of the new event should correspond to the dates of your season.

Reviewer: ```suggestion
The start and end date of the new event should correspond to the start and end dates of your season.
```

Inside the new event, you only need to create a single product which you can call "season ticket". 
Inside that product's settings, head to the "Additional settings" section and look for the option "This product creates a membership of type". 
Select the membership type you just created. 

> By default, the checkbox "The duration of the membership is the same as the duration of the event or event series date" is active, which is fine for our season ticket example, but you might need to unset it and provide custom timing for other ticket types such as week passes.

Reviewer: > By default, the checkbox "The duration of the membership is the same as the duration of the event or event series date" is active, which is fine for our season ticket example, but you might need to unset it and provide custom timing for other ticket types such as week passes.

Schlage vor, das in zwei Sätze aufzuteilen.

> ![](../../assets/screens/products/seasontickets_issue1.png "Season tickets issue 1")

Reviewer: > ![](../../assets/screens/products/seasontickets_issue1.png "Season tickets issue 1")

Bildbeschreibung fehlt

To prevent confusion, it might be useful to turn off ticket downloading at "Settings" → "Tickets" for your new event. 

> That's it, you are now ready to sell season tickets!

Reviewer: > That's it, you are now ready to sell season tickets!

Bin nicht überzeugt, dass dieser Status bereits zu diesem Zeitpunkt in dem beschriebenen Prozess erreicht ist: Es folgen ja die Schritte, aufgrund welcher sich mit den Mitgliedschaften überhaupt etwas anfangen lässt.

This page shows the list of all products. 
Click the :btn-icon:fa3-plus: Create a new product: button. 
Choose a name, a description, a default price and a sales tax and click the :btn:Save: button. 

You can also edit an already existing product by clicking its name or the :btn-icon:fa3-edit:: edit button next to it in the list. 

### Creating and editing quotas 

A quota determines how many instances of your product can be sold. 
Every product has to be part of at least one quota before it becomes available in the shop. 

If you want to create or edit quotas, navigate to :navpath:Event → :fa3-ticket: Products → Quotas:. 
This page shows the list of all quotas for the event as well as the total capacity and how many items are left for each quota. 

Click the :btn-icon:fa3-plus: Create a new quota: button. 
Choose a name and a capacity, check the products to be contained in this quota, and any advanced options, and then click the :btn:Save: button. 

You can also edit an already existing quota by clicking its name or the :btn-icon:fa3-edit:: edit button next to it in the list. 

> ## Applications 

Reviewer: ```suggestion
## Applications 
```

review gerade bis hier; danach wird es strukturell

Navigate to :navpath:Your event → :fa3-check-square-o: Check-in → Check-in lists: and click the :btn-icon:fa3-plus: Create a new check-in list: button. 

Specify a name, but do **not** choose a specific date. 
This check-in list will apply to all dates. 
Open the :btn:Advanced: tab. 

Under "Custom check-in rule", click the :btn-icon:fa3-plus-circle: Add condition: button and select "At least one of the conditions below (OR)" from the dropdown menu. 
Click the :btn-icon:fa3-plus-circle: Add condition: button again and select "All of the conditions below (AND)" from the dropdown menu. 
You now have an OR-bracket with an AND-bracket inside it. 

Click the :btn-icon:fa3-plus-circle: Add condition: button **within the AND-bracket** and select "Product", then "is one of", and then all admission tickets **other than** the season pass. 
Click the :btn-icon:fa3-plus-circle: Add condition: button within the AND-bracket again and select "Current date and time", then "is after", then "Event start". 
Click the :btn-icon:fa3-plus-circle: Add condition: button within the AND-bracket again and select "Current date and time", then "is before", then "Event end". 

> You can leave the "Tolerance (minutes)" fields empty or specify a tolerance of a few minutes for some leniency. 

Author: Ich hatte ganz naiv angenommen, dass eine Toleranz bei "Current date and time is after event start" oder "event admission" immer bedeutet, dass die Toleranz vor dem Veranstaltungsbeginn gilt. Bei "Is before event end" dann umgekehrt (wobei, wann braucht man das?). 

Kommt man unter irgendwelchen Umständen in die Verlegenheit, da negative Zahlen eingeben zu müssen?

Under "Custom check-in rule", click the :btn-icon:fa3-plus-circle: Add condition: button and select "All of the conditions below (AND)" from the dropdown menu. 
Click :btn-icon:fa3-plus-circle: Add condition: again and select "Current date and time", then "is after", then "Event start". 
You can leave the "Tolerance (minutes)" field empty. 

> If you want attendants to be able get in, for example, 10 minutes before the event starts, specify a "Tolerance (minutes)" of `10`. 

Reviewer: ```suggestion
If you want attendees to be able get in, for example, 10 minutes before the event starts, specify a "Tolerance (minutes)" of `10`. 
```
